quirks/websites-with-shared-credential-backends.json

```diff
@@ -242,5 +242,17 @@
     [
         "asiamiles.com",
         "cathaypacific.com"
+    ],
+    [
+        "americastestkitchen.com",
+        "cooksillustrated.com",
+        "cookscountry.com",
+        "onlinecookingschool.com"
+    ],
+    [
+        "americastestkitchen.com",
+        "cooksillustrated.com",
+        "cookscountry.com",
+        "onlinecookingschool.com"
```

Comment:
It looks like you’ve added the group twice! Can you fix that up?

Reply:
thank you -- i have added another commit to correc this